packages/hardhat-core/src/internal/core/errors-list.ts

```diff
@@ -613,8 +613,26 @@ Please double check your task definitions.`,
 What makes these types special is that they can be represented as strings, so you can write them down in the terminal.`,
       shouldBeReported: false,
     },
-    DEPRECATED_TRANSFORM_IMPORT_TASK: {
+    TASK_SCOPE_CLASH: {
```

Comment:
Do you not usually append to the end of the errors list, rather than reassigning error codes?